### PR TITLE
feat: Add comment on PR

### PR DESCRIPTION
**Pull request by damcou**
Add comment logic in the action to add a comment on the PR

### CODE REVIEW

File: .github/workflows/netlify.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ jobs:
           crawler-user-id: ${{ secrets.CRAWLER_USER_ID }}
           crawler-api-key: ${{ secrets.CRAWLER_API_KEY }}
           crawler-api-base-url: 'https://crawler-dev.algolia.com/api/1/'
+          github-token: ${{ secrets.GIT_HUB_TOKEN }}
```
Comment (bodinsamuel) [inline .github/workflows/netlify.yml:32]: mmh I believe github actions comes with a github token, can you check other implementation?

File: package.json [hunk 2]
```diff
@@ -59,7 +59,7 @@
   },
   "dependencies": {
     "@actions/core": "1.4.0",
-    "@actions/github": "5.0.0",
+    "@actions/github": "^5.0.0",
```
Comment (bodinsamuel) [inline package.json:62]: deps are pinned

File: src/index.ts [hunk 3]
```diff
@@ -54,6 +56,34 @@ function getRecordExtractorSource(): string {
 }`;
 }
 
+function addComment(crawlerId: string): void {
```
Comment (bodinsamuel) [inline src/index.ts:59]: This PR should have a comment no?

File: src/index.ts [hunk 4]
```diff
@@ -54,6 +56,34 @@ function getRecordExtractorSource(): string {
...
+    const protocol = pathArray[0];
+    const host = pathArray[2];
+    const baseUrl = `${protocol}//${host}`;
+
+    const message = `<p>Check Crawler: ${baseUrl}/admin/crawlers/${crawlerId}/overview</p>
+    <p>Check Application: https://www.algolia.com/apps/${ALGOLIA_APP_ID}/explorer/browse/${CRAWLER_NAME}</p>`;
+
+    const context = github.context;
+    if (context.payload.pull_request === undefined) {
+      core.setFailed('No pull request found.');
```
Comment (bodinsamuel) [inline src/index.ts:71]: It will fail on master, I don't think comment is a mandatory step, it should just log.

File: src/index.ts [hunk 5]
```diff
@@ -54,6 +56,34 @@ function getRecordExtractorSource(): string {
...
 }
 
+function addComment(crawlerId: string): void {
+  try {
+    const pathArray = CRAWLER_API_BASE_URL.split('/');
+    const protocol = pathArray[0];
+    const host = pathArray[2];
+    const baseUrl = `${protocol}//${host}`;
+
+    const message = `<p>Check Crawler: ${baseUrl}/admin/crawlers/${crawlerId}/overview</p>
```
Comment (bodinsamuel) [inline src/index.ts:66]: Not sure how it renders but I'm sure we can do better.
At least put some proper links ☺️

File: .github/workflows/netlify.yml (hunk 1 above)
Comment (damcou) [reply .github/workflows/netlify.yml:32]: Apparently, this issue has been already raised ;) : https://github.com/peaceiris/actions-gh-pages/issues/309

Comment (damcou) [reply .github/workflows/netlify.yml:32]: gonna set the default as in the example

File: src/index.ts (hunk 3 above)
Comment (damcou) [reply src/index.ts:59]: Actually, the GA tests are running the version 0.6 which doesn't have the comment feature yet. I'm not sure how to test it efficently

**Review by bodinsamuel — APPROVED** [review]
👌🏻

File: src/index.ts [hunk 6]
```diff
@@ -54,6 +56,34 @@ function getRecordExtractorSource(): string {
...
+
+    const context = github.context;
+    if (context.payload.pull_request === undefined) {
+      core.info('No pull request found.');
+      return;
+    }
+    const prNumber = context.payload.pull_request.number;
+
+    const octokit = github.getOctokit(GITHUB_TOKEN);
+    octokit.rest.issues.createComment({
```
Comment (bodinsamuel) [inline src/index.ts:77]: Now that I think about it, would be create to not recreate a comment but find the previous one
can be done in an other PR

**Pull request merged**